lib/getColorForCategoryId.js

```diff
@@ -0,0 +1,13 @@
+const getColorForCategoryId = (categoryId) => {
+  if (categoryId === 1) {
+    return "yellow";
+  } else if (categoryId === 2) {
+    return "green";
+  } else if (categoryId === 3) {
+    return "indigo";
+  } else {
+    return "red";
+  }
+};
```

Comment:
Absolutely fine, may be clearer as a switch statement but it wouldn't change the behaviour